backend/trade/serializers.py

```diff
@@ -56,6 +58,21 @@ def create(self, validated_data):
         trade.recipient_decision = 'pending'
         trade.save()
 
+        new_chat = TradeChat.objects.create(trade_id=trade.id)
+        new_chat.participants.add(initiator, recipient)
+
+        proposed_cash = f'{trade.initiator_cash} cash' if trade.initiator_cash > 0 else '.'
```

Comment:
Unused var?

Reply:
Done. I had forgotten to add it to the message.